NitroxServer/Communication/UdpServer.cs

```diff
@@ -26,13 +26,13 @@ public class UdpServer
         private readonly Thread thread;
         private int PortNumber, MaxConn;
 
-        public UdpServer(PacketHandler packetHandler, PlayerManager playerManager, EntitySimulation entitySimulation, ServerConfigReader ConfigReader)
+        public UdpServer(PacketHandler packetHandler, PlayerManager playerManager, EntitySimulation entitySimulation, ServerConfig ConfigReader)
         {
             this.packetHandler = packetHandler;
             this.playerManager = playerManager;
             this.entitySimulation = entitySimulation;
-            PortNumber = ConfigReader.serverPort;
-            MaxConn = ConfigReader.maxConn;
+            PortNumber = ConfigReader.ServerPort;
```

Comment:
Could you make the `ConfigReader` parameter to `configReader`. The code style was wrong before your commits but since you're touching it.. :)